lib/file_generator.dart

```diff
@@ -363,7 +363,7 @@ class FileGenerator extends ProtobufContainer {
       // Make sure any other symbols in dart:core don't cause name conflicts
       // with enums that have the same name.
       out.println("// ignore: UNUSED_SHOWN_NAME\n"
-          "import 'dart:core' show int, dynamic, String, List, Map;");
```

Comment:
Sorry, remembered a bit wrong: It's the VM and dart2js that disagree on the status of `dynamic`. The VM considered `dynamic` to be a part of `dart:core`, and would fail to find it if we didn't show it (since we implicitly hide everything else from `dart:core`).

Analyzer and dart2js both agree that `dynamic` is a pre-defined thing, and not part of `dart:core`, but they only throw a hint/warning, so we chose the lesser of the two evils.